classes/alertmanager.py

```diff
@@ -47,7 +47,7 @@ def find_channel_by_severity_label(self, alert_severity_label):
     # Find the receiver name for the given severity
     receiver_name = ''
     if self.json_config_data is None:
-      return ''
+      return 'Alertmanager data not available'
```

Comment:
Wouldn't it be better to not call this if we failed to call the alert manager API? 

Could add a 
```
 def isDataUnavailable(self):  self.json_config_data is None
```

and throw an exception here... and only doing anything related to alertmanager data when it is available? 

This might help simplify the code a lot